astropy/utils/tests/test_introspection.py

```diff
@@ -73,14 +73,3 @@ def _minversion_test():
         assert minversion(test_module, version)
     for version in bad_versions:
         assert not minversion(test_module, version)
-
-
-def test_minversion():
```

Comment:
Does `_minversion_test` above still run with this removed?

Reply:
I agree, I think ``_minversion_test`` needs to be renamed to ``test_minversion``

Reply:
Wow, proof yet again how useful review is...  How could I have missed that!